client/post-editor/editor-sharing/publicize-options.jsx

```diff
@@ -203,26 +166,6 @@ class EditorSharingPublicizeOptions extends React.Component {
 			);
 		}
 
-		if ( this.props.site && this.props.site.jetpack && ! this.props.isPublicizeEnabled ) {
-			return (
-				<div className="editor-sharing__publicize-disabled">
```

Comment:
The CSS rules for this class can also be removed. And maybe there's more CSS to remove than this.